src/Client/OrchestrationServiceClientShim/ShimDurableTaskClient.cs

```diff
@@ -169,11 +169,16 @@ public override AsyncPageable<OrchestrationMetadata> GetAllInstancesAsync(Orches
 
     /// <inheritdoc/>
     public override Task TerminateInstanceAsync(
-        string instanceId, object? output = null, CancellationToken cancellation = default)
+        string instanceId, TerminateInstanceOptions? options = null, CancellationToken cancellation = default)
     {
+        if (options is null)
+        {
+            options = new TerminateInstanceOptions();
```

Comment:
Same here as the other comment: avoid unnecessary object allocation

Reply:
Done